query-connector/src/app/database-service.ts

```diff
@@ -688,8 +688,7 @@ export async function getCustomQueries(): Promise<CustomUserQuery[]> {
       q.query_data,
```

Comment:
we can just return all the entries in the query table now that we only seed the table with the five demo queries